<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Browse1</class>
 <widget class="QMainWindow" name="Browse1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查询房间信息</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>查询房间信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>查询指定房间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>输入查询信息：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>输入查询内容：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>540</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>320</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>房间一览：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>房间号</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>房间类型</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>房间面积</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>租金</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>房间状态</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>641</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>381</y>
      <width>631</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
